index.html

```diff
@@ -335,5 +240,6 @@ <h2 class="intro">
         </li>
       </ul>
     </footer>
+    <script type="module" src="/js/script.js"></script>
```

Comment:
- [ ] The script file is not being loaded in the online version of the project and this is causing the disappearance of the projects section as seen below making the project quite hard to review. I suggest that you reference it well like this 👉🏻 `src="./js/script.js` or `src="js/script.js`

![image](https://user-images.githubusercontent.com/4247740/157564999-621b0127-b82b-4505-8542-70c0ae6cc871.png)